kie-soup-maven-utils/kie-soup-maven-integration/src/main/java/org/appformer/maven/integration/InJarArtifactResolver.java

```diff
@@ -96,105 +60,8 @@ private List<URL> buildResources(Predicate<String> predicate) {
         return resources;
```

Comment:
At this point, it might be helpful to log "resources" before returning (especially if it's empty) to find out that path may exist but it's wrong, so no resource was found out, wdyt?